### ISSUE TITLE
Automatic printer output handling

### ISSUE THREAD

**Issue by ribbons** [mattgodbolt/jsbeeb#288]
As an enhancement to #287 - I was wondering if it would be possible to automatically detect writes to the printer and store it so that showing the printer output window with Control-B would show any content written before the window was opened?

This would also have the effect of unblocking some programs which write to both the screen and printer, such as this cheesy poetry: https://bbc.godbolt.org/?embed&disc=https:8bs.nerdoftheherd.com/discimgs/8BS35.dsd&loadBasic=https://8bs.nerdoftheherd.com/8BS35/content/2-coalman/emulate.bas&autorun

**Comment by mattgodbolt** [comment]
Great idea. @scarybeasts any thoughts, seems like this should be a pretty change?

**Comment by scarybeasts** [comment]
Yeah, shouldn't be too tricky. We may want it behind a new URL parameter though, unless we want to emulate a printer present by default?

**Comment by mattgodbolt** [comment]
I'm thinking we should probably pop the printer output open if anything goes to it, honestly. It's probably OK to have one by default?

**Comment by ribbons** [comment]
Awesome, thank you!

**Comment by mattgodbolt** [comment]
Yay :) for 8 year old bugs :)
